zilencer/migrations/0022_remotezulipserver_create_audit_log_backfill.py

```diff
@@ -0,0 +1,48 @@
+from django.db import migrations
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def backfill_remote_zulip_server_creation_log_events(
+    apps: StateApps, schema_editor: DatabaseSchemaEditor
+) -> None:
+    RealmAuditLog = apps.get_model("zerver", "RealmAuditLog")
+    RealmAuditLog.REMOTE_SERVER_CREATED = 10215
+
+    RemoteZulipServer = apps.get_model("zilencer", "RemoteZulipServer")
+    RemoteZulipServerAuditLog = apps.get_model("zilencer", "RemoteZulipServerAuditLog")
+
+    objects_to_create = []
+    for remote_server in RemoteZulipServer.objects.all():
+        entry = RemoteZulipServerAuditLog(
+            server=remote_server,
+            event_type=RealmAuditLog.REMOTE_SERVER_CREATED,
```

Comment:
wouldn't it be more natural to have `RemoteZulipServerAuditLog.REMOTE_SERVER_CREATED` here, given that's the model we're operating on rather than taking the constant from `RealmAuditLog`?

Reply:
Yeah, we should write it that way.